dm/loader/lightning.go

```diff
@@ -329,6 +329,41 @@ func GetLightningConfig(globalCfg *lcfg.GlobalConfig, subtaskCfg *config.SubTask
 	if err := cfg.LoadFromGlobal(globalCfg); err != nil {
 		return nil, err
 	}
+	cfg.TiDB.Security = &globalCfg.Security
+	// TODO: Using TLS content cannot verify certificates correctly when lightning access PD server.
```

Comment:
```suggestion
	// TODO: avoid writing to local file. right now we don't know how to verify certificates correctly using TLS content in a short time, but we have a time schedule to keep.
```
so the reason we write to file is we hasn't found the way to fix this in short time?

Reply:
yes